### DISCUSSION TITLE
How to resize an image in a PictureBox control?

### DISCUSSION THREAD

**Discussion by haplox** [twinbasic/twinbasic#1378] [Q&A]
Hello all,

I have been trying with scalewidth, scaleheight without result. In the image control there is an stretch option, but not in picturebox tha it is what I need.

Thanks in advance

**Comment by haplox** [comment]
Well... as ther is not any answer, finally I have understood how it work (I think). When you create a PictureBox, of any dimenssions, tB asigns a number to ScaleHeight and ScaleWidth. 

If you change the ScaleMode to vbPixels, the value for both scales gives you the size in pixels of your image to fullfill the PictureBox

**Reply by wqweto** [reply]
Btw, do you have experience with VB6 and VBA?

The `PictureBox` control in TB works exactly the same as in VB6/VBA because one of the goals of the project is to be 100% backwards compatible to these legacy COM versions of VB.

If you come from VB.Net there are certainly going to be differences with TB's implementation.
